### DISCUSSION TITLE
Is it possible to support output the number of problems for each rule after verifying all the code?

### DISCUSSION THREAD

**Discussion by Youky1** [eslint/eslint#15751] [Ideas] [answered]
Do you accept pull requests if possible?

**Comment by voxpelli** [comment, marked as answer]
You can use a custom formatter such as [eslint-formatter-summary](https://github.com/mhipszki/eslint-formatter-summary)

Then you'll get an output like:

<img width="715" alt="28670748-ff4cff36-72d1-11e7-8fc0-b0d6a12c69ea" src="https://user-images.githubusercontent.com/34457/170712044-96ce4884-7a3a-495a-8b4d-07eda3d5c2a7.png">
